lib/json_schemer/format.rb

```diff
@@ -101,7 +101,14 @@ def valid_uri_reference?(data)
     end
 
     def iri_escape(data)
-      URI.escape(data, /[^[[:ascii:]]]/)
+      data.gsub(/[^[:ascii:]]/) do |match|
```

Comment:
Interesting, `/[^[:ascii:]]/` looks more correct that what I had before. Do you know if there's a difference between that and `/[^[[:ascii:]]]/`?

Reply:
i don't think so i just think it's less characters